book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Let's add `:rentals` to `attr_accessor` as users can write and read the rentals. See suggestion below:

```suggestion
  attr_accessor :title, :author, :rentals
```

Reply:
This issue is well resolved! ✔️